swagger/src/test/scala/org/scalatra/swagger/SwaggerSpec.scala

```diff
@@ -420,7 +421,7 @@ class SwaggerSpec2 extends ScalatraSpec with JsonMatchers {
     val JObject(definitions) = j
     definitions.map {
       case (name, definition) =>
-        verifyFields(definition, swaggerJsonJValue \ "securityDefinitions" \ name, "type", "name", "in", "flow", "authorizationUrl", "scopes")
+        verifyFields(definition, swaggerJsonJValue \ "securityDefinitions" \ name, "type", "name", "description", "in", "flow", "authorizationUrl", "scopes")
```

Comment:
luckily the `description` field is not part of the [swagger 1.2 spec](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/1.2.md), so no need to implement or test. It is tested in `SwaggerSpec2`. Do you agree?